integration_tests/pages/probationDocuments.ts

```diff
@@ -0,0 +1,42 @@
+import Page, { PageElement } from './page'
+
+export default class ProbationDocumentsPage extends Page {
+  constructor() {
+    super('Documents held by probation')
+  }
+
+  documentsTable = (table: () => PageElement, row: number) => {
+    const tableRow = () => table().find('tbody').find('.govuk-table__row').eq(row)
+    const details = () => tableRow().find('.govuk-table__cell').eq(0)
+    return {
+      details: () => ({
+        documentLink: () => details().find('.govuk-grid-column-full').eq(0).find('a'),
+        documentTypeAndAuthor: () => details().find('.govuk-grid-column-full').eq(1),
+        documentDescription: () => details().find('.govuk-grid-column-full').eq(2),
+      }),
+      date: () => tableRow().find('.govuk-table__cell').eq(1),
+    }
+  }
```

Comment:
These structures are really nice!

Reply:
creating nice page objects is always a satisfying thing to do :')